DuckDuckGo/en.lproj/Localizable.strings

```diff
@@ -151,3 +151,7 @@
 "keyCommandFire" = "Clear all tabs and data";
 "keyCommandClose" = "Close";
 "keyCommandSelect" = "Select";
+
+"bookmarkAll.tabs.notfound" = "No open tabs found to bookmark";
+"bookmarkAll.tabs.saved" = "All open tabs are bookmarked";
+"bookmarkAll.tabs.failed" = "Failed to bookmark %lu out of %lu tab(s)";
```

Comment:
I'm wondering about the exact messages that we want to have here - I will come back to you with it in case there's a need for a change before merging PR.

Reply:
Let's go with:
- No tabs to bookmark
- All tabs bookmarked
- Unable to bookmark some tabs

This way in the case of the failure (that corresponds to the last message), we don't need to return counts at all, just simple bool flag to indicate the problem.

Reply:
👍🏻